### ISSUE TITLE
Logging into api via app on server shows 403 errors 

### ISSUE THREAD

**Issue by jreid06** [directus/api#651]
# Bug Report

## Steps to Reproduce

If you would like I can give you access to the API instance in a private conversation so you can see first hand the problem I am facing. 

## Expected Behavior

I've got the app hosted on my server under domain http://directus.jbreid.co.uk

- When I select the API instance (LIVE PORTFOLIO API) I want to connect to it verifies it and allows me to enter my login details

- I enter the correct credentials and it should take me to my last location e.g http://directus.jbreid.co.uk/#/collections

## Actual Behavior
1.  Login credentials are correctly verified. redirected to http://directus.jbreid.co.uk/#/collections 
2.  HOWEVER I see "Server Error Something is wrong with this instance’s server or database" screen.
3. In the console it tells me that its a 403 (forbidden) error

## Other Context & Screenshots
<img width="1362" alt="screenshot 2018-12-12 at 17 04 26" src="https://user-images.githubusercontent.com/19251469/49885837-29ebba80-fe30-11e8-9565-aa925b2e48c2.png">

## Technical Details
- API Version - version": "2.0.9"
- Device: Mac
- OS: [eg: MacOS 10.13.6, Windows 10.1803]
- Web Server: Apache 2.4.37
- PHP Version:  7.2.0
- Database: MySQL 5.5
- Install Method: cloned `master` branch]

**Comment by WellingGuzman** [comment]
Hey @jreid06,

I have some question and requests:

1. What are the permissions of the user you are using to authenticate?
2. Can you share the database you are using?
3. Can you show us the errors in your logs in the `logs` directory?
4. Can you also try authenticating using the API `/_/auth/authenticate` endpoint and try to use that token to request say: `/_/users`.

**Comment by jreid06** [comment]
Hi @WellingGuzman 

1. I'm the admin user created when installing Directus API to authenticate
2. Im using a MySQL database. Version 5.5. Or did you mean the actual ".sql" file?
3. In the logs file i've added the most recent log from the api error
[logs.txt](https://github.com/directus/api/files/2673497/logs.txt)

4. Authenticating using the API works as expected. 
<img width="381" alt="screenshot 2018-12-12 at 21 10 11" src="https://user-images.githubusercontent.com/19251469/49898982-5ca6aa80-fe52-11e8-8cb1-c3e202c278f0.png">

**More information**

I've noticed another issue. Not sure if its related. 
When I query the API using a php curl library I set the Authorization header with the token e.g "Bearer $token"; However I still get a 403 (forbidden error). It seems as if the header is being ignored. 

When I put the access_token=$token at the end of the GET or POST request the API responds successfully
